src/w2grid.js

```diff
@@ -3446,19 +3446,28 @@
             // insert expand row
             var tmp = 1 + (this.show.selectColumn ? 1 : 0);
             var addClass = ''; // ($('#grid_'+this.name +'_rec_'+ w2utils.escapeId(recid)).hasClass('w2ui-odd') ? 'w2ui-odd' : 'w2ui-even');
+            
             $('#grid_'+ this.name +'_rec_'+ id).after(
                     '<tr id="grid_'+ this.name +'_rec_'+ recid +'_expanded_row" class="w2ui-expanded-row '+ addClass +'">'+
-                        (this.show.lineNumbers ? '<td class="w2ui-col-number"></td>' : '') +
-                    '    <td class="w2ui-grid-data w2ui-expanded1" colspan="'+ tmp +'"><div style="display: none"></div></td>'+
+                        (this.show.lineNumbers ? '<td class="w2ui-col-number"></td>' : '') +                    
                     '    <td colspan="100" class="w2ui-expanded2">'+
                     '        <div id="grid_'+ this.name +'_rec_'+ recid +'_expanded" style="opacity: 0"></div>'+
                     '    </td>'+
                     '</tr>');
+            
```

Comment:
Thanks for fixing expand and collapse. I will merge it, but long term goal is to refactor subgrids in favor of tree like single grid.